### PR TITLE
Fix disk_usage of table_status

### PR DESCRIPTION
**Pull request by yangzb09**
Use sum to calculate total disk usage metrics.

### CODE REVIEW

**Comment by yangzb09** [comment]
image

File: provisioning/dashboards/Dashbase Overview.json [hunk 1]
```diff
@@ -1087,9 +1086,10 @@
       {
         "allValue": null,
         "current": {
-          "text": "All",
+          "tags": [],
+          "text": "intraswitch",
           "value": [
-            "$__all"
+            "intraswitch"
```
Comment (javasoze) [inline provisioning/dashboards/Dashbase Overview.json]: why is this change here?

Comment (yangzb09) [reply provisioning/dashboards/Dashbase Overview.json]: The filter changed when I save dashboard to JSON. I set it back.
